presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -275,7 +275,7 @@ Property Name                                           Description
                                                         is required if the iceberg.catalog.type is ``hadoop``.
                                                         Otherwise, it will be ignored.
 
-``iceberg.file-format``                                 The storage file format for Iceberg tables. The available     ``ORC``
+``iceberg.file-format``                                 The storage file format for Iceberg tables. The available     ``PARQUET``
```

Comment:
I don't understand this, either before or after the PR. "The available PARQUET values are PARQUET and ORC"? That doesn't make sense to me.

Reply:
As I understand, this description means that the value of `iceberg.file-format` could be configured to `PARQUET` or `ORC`, and the default value is `PARQUET`.

Reply:
> I don't understand this, either before or after the PR. "The available PARQUET values are PARQUET and ORC"? That doesn't make sense to me.

@elharo, I believe that's probably because the formatting for the content of the columns of the table does not read in .rst the way that the .html file generated from the .rst file compiles when it is built. 

Here's a screenshot of the current [Iceberg Connector doc](https://prestodb.io/docs/current/connector/iceberg.html#configuration-properties) showing the configuration property: 
<img width="739" alt="Screenshot 2024-07-19 at 11 00 52 AM" src="https://github.com/user-attachments/assets/144de187-2d4e-4d03-ba1f-273bd425f67a">

Here is a screenshot of this PR in a local doc build with the default value corrected from ``ORC`` to ``PARQUET``. 
<img width="962" alt="Screenshot 2024-07-19 at 10 56 15 AM" src="https://github.com/user-attachments/assets/a0ca04dc-71d7-48a9-9f08-e15183f6a17d">

I have found that a good way to avoid this kind of problem when reviewing Presto documentation is to pull the branch and make a local doc build, then review the page in the local doc build. See [Building the documentation](https://github.com/prestodb/presto/tree/master/presto-docs#building-the-documentation) in the presto-docs README for more information.

Reply:
@elharo Sorry for misunderstanding what you meant. As @steveburnett explained, it's the formatting for table columns in `.rst` that caused your confusion.